src/app/modules/registration/pages/ice/ice-thickness/ice-layer/ice-layer.page.ts

```diff
@@ -34,19 +43,23 @@ import { TranslatePipe } from '@ngx-translate/core';
     IonTitle,
     IonToolbar,
     KdvSelectComponent,
-    ModalSaveOrDeleteButtonsComponent,
     NumericInputComponent,
     TranslatePipe,
   ],
 })
+/** Skjema for å registrere et islag */
 export class IceLayerPage {
   private modalController = inject(ModalController);
 
   readonly iceThicknessLayer = input<IceThicknessLayerEditModel>();
   layerTid = linkedSignal(() => this.iceThicknessLayer()?.IceLayerTID);
   thickness = linkedSignal(() => this.iceThicknessLayer()?.IceLayerThickness);
   isValid = computed(() => this.thickness() != null);
-  isNew = computed(() => this.iceThicknessLayer() != null);
+  isNew = computed(() => this.iceThicknessLayer() == null);
```

Comment:
Ser denne linja er endra, regner med det kun er den som egentlig løser feilen - at slettknappen ikke ble vist - eller er det andre grunner til å ikke bruke komponenten som ble brukt før? 

Hvis det er det, bør den komponenten kanskje slettes?

Hvis ikke, kanskje vi heller burde brukt den komponenten også på snøprofil? Only once og så videre.

Reply:
Helt korrekt. Det var unødvendig å kopiere knappene fra snøprofil. Har rullet tilbake den endringen.
Forsøkte å bruke felles-komponenten for ok/slett i snøprofil, men sleit med å få valideringa til å funke slik det er laget. Denne bruker form-validering, mens felles-komponenten kaller en funksjon på click-eventet til OK-knappen. Så da lot jeg det være.